src/Common/StringParsing.fs

```diff
@@ -15,8 +15,12 @@ open FSharp.Formatting.Markdown
 
 module String =
     /// Matches when a string is a whitespace or null
+    [<return: Struct>]
```

Comment:
Do you have any idea under which conditions you would start using this?

- What is the minimum language version?
- Always for unit or can this be used for other return types as well?

Reply:
Struct constructs came with FSharp.Core 6.0.1 so that's the minimum version. 
Works with other return types too.